pkg/dependenciesdistributor/dependencies_distributor.go

```diff
@@ -546,6 +546,7 @@ func (d *DependenciesDistributor) removeScheduleResultFromAttachedBindings(bindi
 		delete(attachedBindings[index].Labels, bindingLabelKey)
 		updatedSnapshot := deleteBindingFromSnapshot(bindingNamespace, bindingName, attachedBindings[index].Spec.RequiredBy)
 		attachedBindings[index].Spec.RequiredBy = updatedSnapshot
+		attachedBindings[index].Spec.ConflictResolution = ""
```

Comment:
Do you need the latest code for rebase?

Reply:
already rebased your PR, waiting for #5731 or #4418 to be merged and do rebase

Reply:
done